Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -51,8 +51,7 @@ struct DashboardView: View {
                     Group {
                         InAppMessageViewRepresentable(elementId: "dashboard-announcement", onActionClick: { _, _, _ in
                             print("Custom callback received")
-                        })
-                        .frame(height: 350)
+                        }).frame(maxWidth: .infinity)
```

Comment:
No longer required for customer to set a hard coded height and width. Instead, customers are required to set a width. 

I found you do need to set the `frame()` but you can use `maxWidth: .infinity` which is setting width to 100% of the parent container. 

I am not yet sure if it will be required in the final public API of the SwiftUI inline View if this is a requirement for customers to set. We do have a requirement in our UIKit View that customers must set a width so I think it's OK to also have this requirement for the SwiftUI View?

Reply:
I think it should be SwiftUI customer;s responsibility to set the frame based on their app requirements, just like we expect from UIKIt customers.

Reply:
Also, would you mind posting a quick video of this change in PR description or share the link if it is already there in any other PR? I am wondering how it would take the shape with infinity width. 🤔

Reply:
No problem. I added a video to description.

Reply:
Thanks for that. :)